packages/react-components/src/components/SegmentedControl/SegmentedControl.stories.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
```

Comment:
Can you follow the story convention from eg. `Avatar`?

We're trying to avoid the renaming of the component using the `Component` suffix. This happens thanks to naming the first story `Default`.